src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -154,6 +156,7 @@ def create_request(
         config
         :param post_filter: Flag for doing either post_filter or regular
         querying (i.e. faceting or not)
+        :param List source_filter: Source Filter (``_source``)
```

Comment:
No need to duplicate type hints in doc strings. Type hints should always be in the signature.

Also, explaining a parameter called `source_filter` with the words "Source Filter" is ironic, don't you think?